### PR TITLE
Doc: Equipment reports

### PR DESCRIPTION
**Pull request by nonifier**
Add API documentation for `/equipment_reports` and the associated object response definitions.

### CODE REVIEW

File: documentation/slate/CONTRIBUTING.md [hunk 1]
```diff
@@ -15,7 +15,12 @@ plus some Slate widgets (such as notices).
 
 Once some changes have been done, Slate needs to build HTML sources from Markdown files.
 
-First, install bundler using `apt-get install bundler` or `gem install bundler`.
+First, install bundler using `apt-get install bundler` or `gem install bundler`. Make sure you've also installed ruby's header to build the json package.
```
Comment (benoit-bst) [inline documentation/slate/CONTRIBUTING.md:18]: Why you need to install ruby to run `/equipment_reports` API ?

File: documentation/slate/source/includes/objects.md [hunk 2]
```diff
@@ -624,6 +624,101 @@ Cities are mainly on the 8 level, dependant on the country
...
+```json
+"stop_area_equipments": [
+    {
+        "equipment_details": [...],
+        "stop_area": {...},
+    },
+    ...
+]
+```
+A list of object that maps foreach stop area, the equipments details.
```
Comment (mbounabi) [inline documentation/slate/source/includes/objects.md]: This isn't C++, "foreach" is actually two words :grin:

File: documentation/slate/source/includes/objects.md [hunk 3]
```diff
@@ -624,6 +624,101 @@ Cities are mainly on the 8 level, dependant on the country
...
+
+|Field|Type|Description|
+|-----|----|-----------|
+|equipment_details|[equipment-details](#equipment-details)|The line to which equipments are associated |
```
Comment (mbounabi) [inline documentation/slate/source/includes/objects.md]: Wrong description (copy-paste...)

File: documentation/slate/CONTRIBUTING.md (hunk 1 above)
Comment (mbounabi) [reply documentation/slate/CONTRIBUTING.md:18]: It's to generate the documentation

File: documentation/slate/source/includes/apis.md [hunk 4]
```diff
@@ -1049,7 +1049,7 @@ In this example, the stop points within the circle (SP1, SP2 et SP3) can be reac
 The journeys can only use allowed vehicle journeys (as present in the `public_transport` or `on_demand_transport` sections).
 They also can only use the allowed stop points for getting in or out of a vehicle (as present in the `street_network`, `transfer` and `crow_fly` sections).
 
-For filtering vehicle journeys, the identifier of a line, route, commercial mode, physical mode or network can be used. 
+For filtering vehicle journeys, the identifier of a line, route, commercial mode, physical mode or network can be used.
```
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: Is that correct? I know you didn't write it but shouldn't it be 'To filter vehicle journeys' ?

File: documentation/slate/source/includes/objects.md [hunk 5]
```diff
@@ -624,6 +624,94 @@ Cities are mainly on the 8 level, dependant on the country
...
+    "updated_at": "2019-05-17T15:54:53+02:00"
+}
+```
+
+|Field|Type|Required|Description|
+|-----|----|--------|-----------|
+status|string|always|Equipment status: <ul><li>`"unknown"`: no realtime information available</li><li>`"unavailable"`: equipment is known to be unavailable with details provided below </li></ul>
+cause|label|optional|If `unavailable`, gives you the cause in a label
+effect|label|optional|If `unavailable`, gives you the effect in a label
+periods|period|optional|If `unavailable`, gives the effected period with a begin & end date
```
Comment (mbounabi) [inline documentation/slate/source/includes/objects.md]: Minor: maybe add -> If status is 'unavailable'

**Review by mbounabi — APPROVED** [review]

**Review by kinnou02 — APPROVED** [review]

File: documentation/slate/source/includes/apis.md (hunk 4 above)
Comment (nonifier) [reply documentation/slate/source/includes/apis.md]:
```suggestion
To filter vehicle journeys, the identifier of a line, route, commercial mode, physical mode or network can be used.
```

File: documentation/slate/source/includes/objects.md [hunk 6]
```diff
@@ -624,6 +624,94 @@ Cities are mainly on the 8 level, dependant on the country
...
+    ],
+    "status": "unavailable",
+    "updated_at": "2019-05-17T15:54:53+02:00"
+}
+```
+
+|Field|Type|Required|Description|
+|-----|----|--------|-----------|
+status|string|always|Equipment status: <ul><li>`"unknown"`: no realtime information available</li><li>`"unavailable"`: equipment is known to be unavailable with details provided below </li></ul>
+cause|label|optional|If `unavailable`, gives you the cause in a label
```
Comment (nonifier) [inline documentation/slate/source/includes/objects.md]:
```suggestion
cause|label|optional|If status is `unavailable`, gives you the cause in a label
```

File: documentation/slate/source/includes/objects.md [hunk 7]
```diff
@@ -624,6 +624,94 @@ Cities are mainly on the 8 level, dependant on the country
...
+effect|label|optional|If `unavailable`, gives you the effect in a label
```
Comment (nonifier) [inline documentation/slate/source/includes/objects.md]:
```suggestion
effect|label|optional|If status is `unavailable`, gives you the effect in a label
```

File: documentation/slate/source/includes/objects.md (hunk 5 above)
Comment (nonifier) [reply documentation/slate/source/includes/objects.md]:
```suggestion
periods|period|optional|If status is `unavailable`, gives the effected period with a begin & end date
```

**Pull request merged**